src/modules/auth/home/HomeScreen.js

```diff
@@ -174,197 +174,214 @@ function HomeScreen(props) {
+											<Col sm={6}>
+												<a className="external-service-title">
+													{t(`HomeScreen|${item.label}`)}
+												</a>
+											</Col>
+											<Col sm={6}>
+												<CustomInput
+													className="float-right"
+													type="switch"
+													defaultChecked={isConnected}
+												/>
+											</Col>
+										</Row>
+									</ListGroupItem>
+								)
+							})}
+
+							<ListGroupItem className="mb-0">
+								<Link to="/change-password" className="d-block">
+									{t('HomeScreen|Password change')}
+								</Link>
+							</ListGroupItem>
+
+							<ListGroupItem className="mb-0">
+								<Row>
+									<Col sm={5}>
+										{updateFeatures.email?.policy === "anybody" ? (
+											<Link to="/manage-email" className="d-block">
+												{t('HomeScreen|Manage email address')}
+											</Link>
+										) : (
+											<div className="text-muted">
+												{t('HomeScreen|Manage email address')}
+											</div>
+										)}
+									</Col>
+									<Col sm={7}>
+										<div
+											className="float-right char-limit char-limit-text"
+											id="emailAddress"
+											name="emailAddress"
+											title={userinfo?.email ?? "" }
+										>
+											{!userinfo?.email || userinfo?.email === "" ?
+												'N/A'
+												:
+												userinfo?.email
+											}
+										</div>
+									</Col>
+								</Row>
+							</ListGroupItem>
+
+							<ListGroupItem className="mb-0">
+								<Row>
+									<Col sm={6}>
+										{updateFeatures.phone?.policy === "anybody" ? (
+											<Link to="/manage-number" className="d-block">
+												{t('HomeScreen|Manage phone number')}
+											</Link>
+										) : (
+											<div className="text-muted">
+												{t('HomeScreen|Manage phone number')}
+											</div>
+										)}
+									</Col>
+									<Col sm={6}>
+										<div
+											className="float-right text-muted pointer char-limit char-limit-number"
+											id="phoneNumber"
+											name="phoneNumber"
+											title={userinfo?.phone_number ?? ""}
+										>
+											{!userinfo?.phone_number || userinfo?.phone_number === "" ?
+												'N/A'
+												:
+												userinfo?.phone_number
+											}
+										</div>
+									</Col>
+								</Row>
+							</ListGroupItem>
+
+							<ListGroupItem className="mb-0">
+								<Row className="align-items-center">
+									<Col sm={6}>
+										<Link to="/manage-totp" className="d-block">
+											{t('HomeScreen|Manage OTP')}
+										</Link>
+									</Col>
+									<Col sm={6}>
+										<div
+											className="float-right text-muted pointer"
+											id="totp"
+											name="totp"
+										>
+											{(userinfo?.available_factors && userinfo.available_factors.indexOf("totp") != -1) ?
+												<div className="status-circle status-green" title={t("HomeScreen|Added OTP")}/>
```

Comment:
@Aringoaway This is not correct, why dont you use just `Active` / `Inactive` instead of `Added OTP` etc... ? It should just indicate, that something is active or not